### PR TITLE
Proxy rewrite

### PR DESCRIPTION
**Pull request by RadekMolenda**
**This is a breaking change**
- Stop using old url and start using proxy url
- unify the bearer client interface:
  - deprecate integartion_host & timeout
  - prefer using host & http_client_settings

### CODE REVIEW

File: bearer/__init__.py [hunk 1]
```diff
@@ -1,109 +1,75 @@
...
-      >>> bearer.invoke('<buid>', 'defaultFunction')
+      >>> bearer.integration('<integration_id>')
     """
 
-    def __init__(self, api_key: str, integration_host: str = PRODUCTION_INTEGRATION_HOST, timeout: int = TIMEOUT):
+    def __init__(
+            self,
+            api_key: str,
+            integration_host: str = None,
+            timeout: int = None,
```
Comment (cfabianski) [inline bearer/__init__.py]: Do we want to keep this one too?

Comment (RadekMolenda) [reply bearer/__init__.py]: I was thinking about backward compatibility

Comment (RadekMolenda) [reply bearer/__init__.py]: we are removing custom function related stuff, but for the proxy part I think it should be fine to keep the deprecation for a while (even though the `timeout` parameter has been introduced very recently)

Comment (cfabianski) [reply bearer/__init__.py]: Up to you but I think it's fine to remove it as it's been introduced only very recently.
Let's remove it at least from the doc / Readme and let's have a deprecation warning if we keep it.

Also, major release == breaking changes ;)

Comment (RadekMolenda) [reply bearer/__init__.py]: https://github.com/Bearer/bearer-python/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R45-R55

File: tox.ini [hunk 2]
```diff
@@ -4,7 +4,4 @@ envlist = py37
 
 [testenv]
 # install pytest in the virtualenv where commands will be executed
-deps = -rrequirements.txt
-commands =
-    # NOTE: you can run any command line tool here - not just tests
-    pytest
\ No newline at end of file
+commands = python setup.py test
```
Comment (didroe) [inline tox.ini]: ❤️

**Pull request merged**